mobile/lib/common/settings_force_close.dart

```diff
@@ -0,0 +1,159 @@
+                        children: [
+                          GestureDetector(
+                            child: const Icon(
+                              Icons.arrow_back_ios_new_rounded,
+                              size: 22,
+                            ),
+                            onTap: () {
+                              context.pop();
+                            },
+                          ),
+                        ],
+                      ),
+                      const Expanded(
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Text(
+                              "Force-Close Channel ",
+                              style: TextStyle(
+                                  fontWeight: FontWeight.w500, fontSize: 20),
+                            ),
+                          ],
+                        ),
+                      )
+                    ],
+                  ),
+                  const SizedBox(
+                    height: 20,
+                  ),
+                  Container(
+                    padding: const EdgeInsets.all(10),
+                    decoration:
+                        BoxDecoration(borderRadius: BorderRadius.circular(15)),
+                    child: RichText(
+                      text: const TextSpan(
+                        style: TextStyle(
+                            fontSize: 18,
+                            color: Colors.black,
+                            letterSpacing: 0.4),
+                        children: [
+                          TextSpan(
+                            text: "Warning",
+                            style: TextStyle(
+                                color: Colors.red, fontWeight: FontWeight.w600),
+                          ),
+                          TextSpan(
+                            text:
+                                ": Force-closing your channel should only be considered as a last resort if 10101 is not reachable.\n\n\n",
+                          ),
+                          TextSpan(
+                              text:
+                                  "It's always better to collaboratively close as it will also save transaction fees.\n\n"),
+                          TextSpan(text: "If you "),
+                          TextSpan(
+                              text: "force-close",
+                              style: TextStyle(
+                                  color: Colors.red,
+                                  fontWeight: FontWeight.w600)),
+                          TextSpan(
+                              text:
+                                  ", you will have to pay the fees for going on-chain.\n\n"),
+                          TextSpan(
+                              text:
+                                  "Your funds can be claimed by your on-chain wallet after a X blocks.\n\n"),
```

Comment:
We will update this with more details later. For now let's just leave it as this.

```suggestion
                                  "Your funds can be claimed by your on-chain after a while.\n\n"),
```